dictionaries/en-common-misspellings/src/fromCrateTypos/softwareTerms.txt

```diff
@@ -916,6 +916,7 @@ thex->then
 thime->time
 threah->thread
 tiem->time
+tiemstep->timestep
```

Comment:
@kenji-miyake,

Did you add this one? This file is auto-generated and will get overwritten.

It is better to add it here: https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/cspell-corrections.yaml